docs/Makefile.am

```diff
@@ -25,16 +25,15 @@
 AUTOMAKE_OPTIONS = foreign no-dependencies
 
 if BUILD_DOCS
-# if we disable man page building, ignore these
-MK_CA_DOCS = mk-ca-bundle.1
+EXTRA_DIST = mk-ca-bundle.1
+all-local: $(MK_CA_DOCS)
 CURLCONF_DOCS = curl-config.1
 endif
 
 
 # EXTRA_DIST breaks with $(abs_builddir) so build it using this variable
 # but distribute it (using the relative file name) in the next variable
 man_MANS = $(abs_builddir)/curl.1
-noinst_man_MANS = $(MK_CA_DOCS)
 dist_man_MANS = $(CURLCONF_DOCS) $(MK_CA_DOCS)
```

Comment:
We don't want to install `mk-ca-bundle.1`, so `$(MK_CA_DOCS)` should be removed here. It will be built by `all-local`, and distributed via `EXTRA_DIST`.

Also, distributing and installing `curl-config.1` should be done always, so we can replace `$(CURLCONF_DOCS)` here with `curl-config.1`.